src/software/jetson_nano/thunderloop.cpp

```diff
@@ -7,39 +7,43 @@
 #include "software/jetson_nano/primitive_executor.h"
 #include "software/jetson_nano/services/motor.h"
 #include "software/logger/logger.h"
+#include "software/logger/network_logger.h"
 #include "software/util/scoped_timespec_timer/scoped_timespec_timer.h"
 #include "software/world/robot_state.h"
 
+// 50 millisecond timeout on receiving primitives before we emergency stop the robots
+const double PRIMITIVE_MANAGER_TIMEOUT_NS = 50.0 * MILLISECONDS_PER_NANOSECOND;
+
+const std::string CPU_TEMP_FILE_PATH = "/sys/class/thermal/thermal_zone0/temp";
```

Comment:
i think thermal_zone1 has cpu temperature. to verify you can check `"/sys/class/thermal/thermal_zone0/type"`

Reply:
Ah good catch, thermal zone 0 was giving me ~50 degrees so I thought it was correct

in order its 

AO-therm
CPU-therm
GPU-therm
PLL-therm
PMIC-Die
thermal-fan-est
iwlwifi

I've changed it to CPU temp thermal zone 1